deploy/static/js/app.js

```diff
@@ -1,5 +1,25 @@
-$(function() {
-  $( '.js-focus' ).focus();
+// Adds a page load event that replaces "img.js-svg"-tags by an embedded version of the image.
+// As fetch()
+window.addEventListener(
+	"load",
+	() => {
+		let domParser = new DOMParser();
+		let embedSvgs = document.querySelectorAll("img.js-svg");
 
-  // Insert your application code here!
-});
+		for( let embedSvg of embedSvgs ) {
+			fetch(embedSvg["src"])
+				.then(response => response.text())
+				.then(svg => {
+					if( !svg )
+						return;
+
+					var svgElement = domParser.parseFromString(svg, "image/svg+xml");
+					svgElement = svgElement.querySelector("svg");
```

Comment:
Are you sure we need to do this? I think the gulp-task will do this, so we don't need to waste resources with parsing on the client...

Reply:
Discussed in a private conversation. Not necessary with the predefined base-structure, but it's the safer way.